lib/manager/npm/update/locked-dependency/parent-version.ts

```diff
@@ -0,0 +1,91 @@
+import { GetPkgReleasesConfig, getPkgReleases } from '../../../../datasource';
+import { logger } from '../../../../logger';
+import { api as semver } from '../../../../versioning/npm';
+
+// Finds the first stable version of parentName after parentStartingVersion which either:
+// - depends on targetDepName@targetVersion or a range which it satisfies, OR
+// - removes the dependency targetDepName altogether, OR
+// - depends on any version of targetDepName higher than targetVersion
```

Comment:
```suggestion
/**
 * Finds the first stable version of parentName after parentStartingVersion which either:
 * - depends on targetDepName@targetVersion or a range which it satisfies, OR
 * - removes the dependency targetDepName altogether, OR
 * - depends on any version of targetDepName higher than targetVersion
 */
```

So it will be shown as function description